src/highlight/HighlightAnnotations.tsx

```diff
@@ -58,6 +62,10 @@ const HighlightAnnotations = (props: Props): JSX.Element => {
 
     const handleCancel = (): void => {
         resetCreator();
+
+        if (isPromoting) {
```

Comment:
Is this check needed? It seems like we could just fire the action, regardless.

Reply:
Yeah the check is not needed for now. It's actually needed for resetting mode, which will be added in the following PR. Don't want to remove it now and add it again later.